index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html> 
+
+<head>
+	<meta name = "viewport" content ="width=device-width, initial-scale=1.0">
+	<meta charset = "UTF-8">
+
+	<title> CSCI 39548 - Assignement 1 </title>title>
+	<link rel ="stylesheet" type="text/css" href="CSS/asyastyle.css">
+</head>
+
+<body>
+	<h1> Animals share their favorite animals, #s, colors </h1>
+	<div class="link">
+		<a href="asya.html"> Asya</a>
+		<a href="momo.html"> Momo </a>
```

Comment:
if @kisa-sou link is above mine she may be happier about this project?..
```suggestion
<a href="kisa.html"> Kisa </a>
<a href="momo.html"> Momo </a>```

Reply:
It does look better this way